parent/pom.xml

```diff
@@ -128,8 +128,8 @@
         <spark.hadoop.version>2.7.7</spark.hadoop.version>
         <spark.version>2.3.3</spark.version>
         <spring.boot.version>1.5.20.RELEASE</spring.boot.version>
-        <spring.data.version>1.13.14.RELEASE</spring.data.version> <!-- don't forget to update spring version -->
-        <spring.version>4.3.25.RELEASE</spring.version><!-- don't forget to update spring-data version -->
+        <spring.data.version>1.13.23.RELEASE</spring.data.version> <!-- don't forget to update spring version -->
+        <spring.version>4.3.29.RELEASE</spring.version><!-- don't forget to update spring-data version -->
         <spring.data-2.0.version>2.0.9.RELEASE</spring.data-2.0.version> <!-- don't forget to update spring-5.0 version -->
         <spring-5.0.version>5.0.17.RELEASE</spring-5.0.version><!-- don't forget to update spring-data-2.0 version -->
```

Comment:
Should we upgrade spring-5.0.version? According https://snyk.io/vuln/SNYK-JAVA-ORGSPRINGFRAMEWORK-1009832 5.0.17.RELEASE is affected too

Reply:
Updated spring 5.